libs/k8s-client/src/ApiClient/StreamClient.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\K8sClient\ApiClient;
+
+use GuzzleHttp;
+use KubernetesRuntime\Authentication;
+use KubernetesRuntime\Exception\CommonException;
+use KubernetesRuntime\Guzzle\Middleware;
+use KubernetesRuntime\Logger;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Log\LogLevel;
+
+class StreamClient
```

Comment:
Nebylo by lepsi jen podedit z toho normalniho clienta a jen
* po volani `parent::__constructor(...)` vyhodit ten nechtenej middleware
* pretizit `configure` aby vracelo `StreamClient`

Jinak mi to totiz prijde 1:1 s tim puvodnim klientem a prijde mi zbytecny mit reimplementovany streba ten auth setup. Navic, na vsechno tady by bylo dobry mit testy.